packages/flutter/lib/src/widgets/text.dart

```diff
@@ -677,6 +682,7 @@ class Text extends StatelessWidget {
             text: data,
             children: textSpan != null ? <InlineSpan>[textSpan!] : null,
           ),
+          copyInterceptor: copyInterceptor,
```

Comment:
I have slight reservations about adding this parameter to every relevant widget. I guess it's not a lot of them, though, and I don't have a better alternative off the top of my head.